ramls/types/resources/resource.json

```diff
@@ -16,7 +16,7 @@
       "description": "List of included items",
       "items": {
         "type": "object",
-        "$ref": "resourceIncludedItem.json"
+        "javaType" : "java.lang.Object"
```

Comment:
A little confused by this change..resourceIncludedItem.json has all possible fields that can be included for a provider, package or title. Why could we just not use that?